src/index.ts

```diff
@@ -21,6 +21,10 @@ function fromEntries(iterable: [string, string][]): {[key: string]: string} {
     .reduce((obj, { 0: key, 1: val }) => Object.assign(obj, { [key]: val }), {})
 }
 
+export interface DependencyLoc {
+  depLoc?: string,
+  styleLoc?: string
```

Comment:
here and everywhere, lets use the word "static" instead of "style", since we're essentially adding support for any non-js asset

Reply:
`static` is reserved, so I'm using `asset`.